### ISSUE TITLE
Resetting the space heating sliders in a new scenario affects the dashboard items

### ISSUE THREAD

**Issue by redekok** [quintel/etmodel#3065]
When a new scenario is created (and no sliders have been touched yet), ... 
![image](https://user-images.githubusercontent.com/5946498/60111543-6f3d5d00-976e-11e9-9d07-0453860052de.png)
... resetting the space heating sliders results in a change in the dashboard items.
![image](https://user-images.githubusercontent.com/5946498/60111781-dfe47980-976e-11e9-8021-e7124c763f6b.png)
This is the case for both PRO and BETA.

**Comment by michieldenhaan** [comment]
This is because we merged the separate heating/hot water sliders in the front-end while leaving them separate in the back-end. The sliders show the starting value of the original space heating sliders. The starting value of the original hot water sliders may be different. Hitting reset sets them equal, resulting in a change in energy flows.

Since the differences are small and a fix is probably not very easy we decided to leave this for now. To fix this before the deploy is a bit ambitious I think 🐵
See [Basecamp](https://3.basecamp.com/3797409/buckets/9242423/messages/1438363512) for some more background.
